setup.py

```diff
@@ -0,0 +1,32 @@
+import os
+import setuptools
+
+with open("README.md", "r") as readme:
+    long_description = readme.read()
+
+deps = os.popen('pip freeze').readlines()
+# readlines() doesn't strip trailing newlines
+deps = [d.strip() for d in deps]
+
+setuptools.setup(
+    name="net-analysis",
+    version="0.0.1",
+    author="Jigsaw Operations, LLC",
+    author_email="fortuna@google.com",
+    description="Network censorship detection tools",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/Jigsaw-Code/net-analysis",
+    packages=setuptools.find_packages(),
+    classifiers=[
+        "Programming Language :: Python :: 3.7",
```

Comment:
I'm going off of the examples at https://packaging.python.org/guides/distributing-packages-using-setuptools/#classifiers where they list individual versions.  These are used for metadata tags in PyPI

Reply:
It's redundant, since we already have the `python_requires` below. I believe the main use is to tell Py2 and Py3 support. I think it would be a pain to maintain such list.

Reply:
Done in 8d66cb7